lib/Sema/TypeCheckDeclOverride.cpp

```diff
@@ -734,6 +734,19 @@ static bool isNSObjectHashValue(ValueDecl *baseDecl) {
   return false;
 }
 
+/// Returns true if the given declaration is for the `NSObject.hash(into:)`
+/// function.
+static bool isNSObjectHash(ValueDecl *baseDecl) {
```

Comment:
The comment states(also the implementation does) that is checking for `NSObject.hash(into:)` but looking the name of the method we cannot distinct if is for `NSObject.hash(into:)` or `NSObject.hash` property. WDYT, does a more specific name make sense? Maybe `isNSObjectHashMethod`?

Reply:
I think `isNSObjectHashMethod` sounds better! Thanks :)